docs/API.md

```diff
@@ -42,18 +42,15 @@ const app = express()
 
 app.use((req, res, next) => {
   const span = tracer.startSpan('web.request')
-  const scope = tracer.scopeManager().activate(span)
-
-  next()
+  
+  tracer.scope().activate(span, () => next())
```

Comment:
Can we add comments to this example, like:

```js
// Start a new span
const span = tracer.startSpan('web.request')

// Execute `next()` within the context of `span` (or w/e is appropriate here)
tracer.scope().activate(span, () => next())
```

Reply:
I'm actually rewriting the docs completely, which I will submit as another PR.